types/types.go

```diff
@@ -0,0 +1,5 @@
+package types
```

Comment:
under types/messages.go you have an interface called Root. 
No need to duplicate it

Reply:
got u.
we will use existing interface but moving to independent file, since it used in qbft/ssv packages as well and not for types/messages only